### ISSUE TITLE
Background conflicts with Cursorline

### ISSUE THREAD

**Issue by brianclements** [Yggdroot/indentLine#125]
I use both cursorline and cursorColumn and there seems to be a clearing out of the background when the indentLine character meets the cursorline, but not the cursorColumn.

![screenshot from 2015-06-06 22 02 03](https://cloud.githubusercontent.com/assets/1616743/8022689/6c8a528a-0c98-11e5-9825-3cb50ba47494.png)

**Comment by magnunleno** [comment]
I was facing the same problem. I checked the docs and found this:

    let g:indentLine_noConcealCursor=1

Problem solved for me. You should give it a try too.

**Comment by brianclements** [comment]
nice suggestion, that almost worked....I guess it's better then before, but I'd still like to have it show up there if possible.

![screenshot from 2015-08-12 10 10 50](https://cloud.githubusercontent.com/assets/1616743/9230991/7849e5a6-40da-11e5-87d4-9654bea74eda.png)

**Comment by mendlik** [comment]
In 06cd61cf7b67e183f84e8a04a89937eb960e9cdb this option was changed to:
```
let g:indentLine_concealcursor=0
```

**Comment by baco** [comment]
Instead of disappearing the the indent line using `g:indentLine_concealcursor=0`, Is it possible to just correct the background color of the indent line character to accommodate to CursorLine/CursorColumn's color?

**Comment by blark** [comment]
Yeah, it would be very nice if the background color could match the cursorline instead of disappearing the indent character. Thanks!

**Comment by Luxed** [comment]
Would that be possible to add a `highlight` value that you cna put in your .vimrc or in your theme directly ? From what I understand, everything that has `ctermbg` will inherit.
I don't understand how this would be difficult, but I don't know anything about vimscript either :/

**Comment by sriharshachilakapati** [comment]
I tried setting this NVIM and couldn't get it to work.

![image](https://user-images.githubusercontent.com/4593204/46781161-ec8a8400-cd3d-11e8-96b8-1d87b314bbf7.png)

I tried both `let g:indentLine_concealCursor=0` and also with c as small letter.

**Comment by fgimian** [comment]
Same little problem here too, it seems we either have to live with a background colour that doesn't match the cursorline:

e.g.

![screen shot 2019-02-04 at 9 37 47 pm](https://user-images.githubusercontent.com/1811813/52203349-253e0280-28c5-11e9-95e3-d8a0e96c7bc6.png)

Or we lose the indent guides all together on the cursorline but retain the colour:

![screen shot 2019-02-04 at 9 37 21 pm](https://user-images.githubusercontent.com/1811813/52203368-3129c480-28c5-11e9-8228-dc9dd325b694.png)

I think it would be awesome to be able to retain the background colour of the cursorline but see the indent guide through it.

Here's an example in Sublime Text for inspiration:

![screen shot 2019-02-04 at 9 39 26 pm](https://user-images.githubusercontent.com/1811813/52203461-61716300-28c5-11e9-8e1d-e0a01f428575.png)

Huge thanks for this awesome plugin 😄 
Fotis

**Comment by baco** [comment]
In NeoVim works fine :slightly_smiling_face:

**Comment by ozars** [comment]
Is there a particular vim issue tracking this problem on upstream?
